Automattic/studio
# Studio: Add translation updates for 1.3.0

## Related issues

Related to https://studioapp2.wordpress.com/2024/11/27/studio-v1-3-0-release/

## Proposed Changes

This PR adds newly translated strings to Studio.

## Testing Instructions



* Pull the changes from this branch
* Start the app with `npm install && npm start`
* Try the app in different languages and confirm that it works as expected 

## Pre-merge Checklist
